### PR TITLE
updating postgresql and mysql tests

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

File: tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+- name: Prepare random number
+  set_fact:
```
Comment (yungezz) [inline tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml]: how about combine with mysql test as alias?

File: tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+  run_once: yes
+
+- name: Create MySQL Server
+  azure_rm_mysqlserver:
+    resource_group: "{{ resource_group }}"
+    name: mysqlsrv{{ rpfx }}
```
Comment (yungezz) [inline tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml]: define whole `mysqlsrv{{ rpfx }}` as a var?

File: tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    name: event_scheduler
+    value: "ON"
+    source: user-override
+  register: output
+- name: Assert the resource instance is well created
+  assert:
+    that:
+      - output.changed
+
+- name: Create again instance of Configuration
```
Comment (yungezz) [inline tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml]: any udpate test?

File: tests/integration/targets/azure_rm_mysqlconfiguration_facts/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+- name: Prepare random number
```
Comment (yungezz) [inline tests/integration/targets/azure_rm_mysqlconfiguration_facts/tasks/main.yml]: add as  alias? why separate those out?

File: tests/integration/targets/azure_rm_postgresqlconfiguration/tasks/main.yml [hunk 5]
```diff
@@ -0,0 +1,63 @@
+- name: Prepare random number
+  set_fact:
+    rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
```
Comment (yungezz) [inline tests/integration/targets/azure_rm_postgresqlconfiguration/tasks/main.yml]: basically same comments as mysql:
1. combine xxxconfiguration test as alias of main module
2. update teste
3.  var definition

File: tests/integration/targets/azure_rm_postgresqlconfiguration/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- name: Create PostgreSQL Server
+  azure_rm_postgresqlserver:
+    resource_group: "{{ resource_group }}"
+    name: postgresqlsrv{{ rpfx }}
+    sku:
+      name: B_Gen5_1
+      tier: Basic
+    location: eastus
+    enforce_ssl: True
+    admin_username: zimxyz
```
Comment (yungezz) [inline tests/integration/targets/azure_rm_postgresqlconfiguration/tasks/main.yml]: more  general  name, and in vars?

File: tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml (hunk 1 above)
Comment (zikalino) [reply tests/integration/targets/azure_rm_mysqlconfiguration/tasks/main.yml]: well, actually I just fixed sku and readded the tests to preview modules, but I can still improve them ;-)

**Comment by zikalino** [comment]
@yungezz they should be, I can make this in the next pr :-)

**Pull request merged**